Tools-Override/tests.targets

```diff
@@ -246,10 +246,8 @@
   </Target>
 
   <Target Name="CheckTestPlatforms">
-    <GetTargetMachineInfo Condition="'$(TargetOS)' == ''">
-      <Output TaskParameter="TargetOS" PropertyName="TargetOS" />
-    </GetTargetMachineInfo>
     <PropertyGroup>
+      <TargetOS Condition="'$(TargetOS)' == ''">$(DefaultOSGroup)</TargetOS>
```

Comment:
I think it's fine to keep them. There are a couple of tests that have `[PlatformSpecific(TestPlatforms.FreeBSD)]`, it's just that we do not have any runs that would ever use that right now. If you were trying to run those tests, you would just need to specify `TargetOS=FreeBSD` manually.